src/platform.ts

```diff
@@ -128,6 +128,12 @@ export class IKHomeBridgeHomebridgePlatform implements DynamicPlatformPlugin {
 
       this.axInstance.get(command).then((res) => {
         res.data.items.forEach((device) => {
+          if (this.config.IgnoreDevices && this.config.IgnoreDevices.find(d => d.toLowerCase() === device.label.toLowerCase())) {
+            this.log.info(`Ignoring ${device.label} becasue it is in the Ignore Devices list`);
+            return;
```

Comment:
I don't think we want to do a return.  We are in a loop and need to continue on with the next device.  We need to ultimately get to the end of the loop and do the resolve.

Reply:
Never mind.  I was wrong.  Tested fine.